### ISSUE TITLE
Saab 340

### ISSUE THREAD

**Issue by Alexmar24** [skiselkov/BetterPushbackC#124]
Won't turn with SAAB 340 in latest version

**Comment by skiselkov** [comment]
Are you testing on XP11 or XP10?

**Comment by birdydma** [comment]
Have you removed the chocks?

Works for me (for me XP 10 only with this plane, BP 0.44).

PS; Saso, set parking brake is not requested if it is already on.
![2017-08-26_13h01_40](https://user-images.githubusercontent.com/30256581/29740800-4d3d34fa-8a5f-11e7-99c5-aafe1f28eabf.jpg)

**Comment by Alexmar24** [comment]
XP11, yes all items removed.

**Comment by skiselkov** [comment]
@birdydma Thanks for the update on the Saab 340 parking brake.
@Alexmar24 I'll need to run some tests to see if I can reproduce this. In the mean time, did you check that you have nosewheel steering enabled? (You need to push the steering tiller in.)

**Comment by Alexmar24** [comment]
Yep, tried with both options (tiller on and tiller off) neither turned

**Comment by birdydma** [comment]
Engines running or not, NWS engaged or not, the turn is done.

Engines running, i have the "You may start engines"

**Comment by Alexmar24** [comment]
Well, not for me

**Comment by birdydma** [comment]
XP 11 is not (yet) a good idea with this plane. Wait for the 1.5 release.

If you have XP 10, can you do the test with it?

**Comment by JetNoise61** [comment]
@skiselkov Hello Sasso, any news on this bug-(fix)
I know, you are busy with the TBM ;-)
Regards 
Oliver
